charts/mattermost-enterprise-edition/templates/service-account.yaml

```
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ .Values.serviceAccount.name }}
-  namespace: {{- .Release.Namespace -}}
+  namespace: {{ .Release.Namespace -}}
```

Just to avoid confusion in the future, I would make it consistent with other values.
```suggestion
  namespace: {{ .Release.Namespace }}
```